libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -126,6 +126,15 @@ class YulUtilFunctions
 	/// signature: (start, end)
 	std::string clearStorageRangeFunction(Type const& _type);
```

Comment:
Needs documentation change.

Reply:
Removing the "only value types" part?

Reply:
Removed that part now. Please clarify if you meant something else.